LobbyServer2/LobbyServer/LobbyServerProtocol.cs

```diff
@@ -907,6 +907,11 @@ public void HandleGroupChatRequest(GroupChatRequest request)
             }
         }
 
+        public void HandleRejoinGameRequest(RejoinGameRequest request)
+        {
+            Send(new RejoinGameResponse() { ResponseId = request.RequestId });
```

Comment:
Always fails?

Reply:
I don't know how to recreate a disconnection other than restart the server, in that case i saw that it expects a response to that, but when i restart the lobby, the game server closes as it lost connection, so i couldn't reconnect properly. but i thought that it may work, or if it doesn't i know that i have to improve this method a little more

Reply:
You can pause your client execution with debugger